pipeline/pipelines/core_generation_pipeline.py

```diff
@@ -16,31 +16,33 @@
    language contains the well known types, defined by protobuf, for that
    language."""
 
-from pipeline.pipelines import code_generation_pipeline
+from pipeline.pipelines import code_generation_pipeline as code_gen
 from pipeline.pipelines import grpc_generation_pipeline
 from pipeline.tasks import protoc_tasks
-from pipeline.tasks import publish_tasks
 
 
-class JavaCoreProtoPipeline(grpc_generation_pipeline.GrpcClientPipeline):
+class CoreProtoPipeline(code_gen.CodeGenerationPipelineBase):
+
+    def __init__(self, **kwargs):
+        super(CoreProtoPipeline, self).__init__(
+            get_core_task_factory(kwargs['language']), **kwargs)
+
+
+class JavaCoreTaskFactory(grpc_generation_pipeline.GrpcTaskFactoryBase):
```

Comment:
Should all of the language-specific task factories be underscored? (are they being used outside of their respective files?)